spyder/plugins/editor/widgets/editor.py

```diff
@@ -2673,6 +2693,10 @@ def __init__(self, parent, plugin, menu_actions, first=False,
                         Defaults to plugin.unregister_editorstack() to
                         unregister the EditorStack with the Editor plugin.
         """
+        for action in menu_actions:
+            if isinstance(action, QAction):
+                print(action.text())

```

Comment:
This `print` is making our tests to fail. @juanis2112, could you remove it? Or is it really necessary to have it?